python/cudf/cudf/core/frame.py

```diff
@@ -327,6 +327,68 @@ def notna(self):
         """
         return self.notnull()
 
+    def interleave_columns(self):
+        """
+       Interleave Series columns of a table into a single column.
+
+        Converts the column major table `cols` into a row major column.
+        Parameters
+        ----------
+        cols : input Table containing columns to interleave.
+
+        Example
+        -------
+        >>> df = DataFrame([['A1', 'A2', 'A3'], ['B1', 'B2', 'B3']])
+        >>> df
+        0    [A1, A2, A3]
+        1    [B1, B2, B3]
+        >>> df.interleave_columns()
+        0    A1
+        1    B1
+        2    A2
+        3    B2
+        4    A3
+        5    B3
+
+        Returns
+        -------
+        The interleaved columns as a single column
+        """
+        result = self._constructor_sliced(
+            libcudfxx.reshape.interleave_columns(self)
+        )
+        result._copy_categories(self)
+        return result
+
+    def tile(self, count):
+        """
+        Repeats the rows from `self` DataFrame `count` times to form a
+        new DataFrame.
+
+        Parameters
+        ----------
+        self : input Table containing columns to interleave.
+        count : Number of times to tile "rows". Must be non-negative.
+
+        Example
+        -------
+        >>> df  = Dataframe([[8, 4, 7], [5, 2, 3]])
+        >>> count = 2
+        >>> df.tile(df, count)
+           0  1  2  3  4  5
+        0  8  4  7  8  4  7
+        1  5  2  3  5  2  3
```

Comment:
Does this tile horizontally or vertically? Based on the test below it looks to be vertically, but this example is showing horizontally.

Reply:
fixed